README.md

```diff
@@ -11,49 +11,373 @@
+      client.query(Create(ref: Ref("keys"), params: Obj(["database": Ref("databases/\(dbName)"), "role": "server"]))) { createKeyR in
+          guard let result = try? createKeyR.dematerialize() else {
+              callback(createKeyR)
+              return
+          }
+          // get the new secret key provided by fauna
+          let newSecret: String = try! result.get(path: "secret")
+          // set up the new client to use the new key
+          client = Client(secret: newSecret, observers: [Logger()])
+          // create a new class into dbName database
+          client.query(Create(ref: Ref("classes"), params: Obj(["name": "posts"]))) { createClassR in
+              guard let _ = try? createClassR.dematerialize() else {
+                  callback(createClassR)
+                  return
+              }
+              // create some indexes to be able to perform queries
+              client.query(
+                Do(exprs:
+                          Create(ref: Ref("indexes"),
+                              params:Obj(["name": "posts_by_tags",
+                                          "source": BlogPost.classRef,
+                                          "terms": Arr(Obj(["field": Arr("data", "tags")])),
+                                          "values": Arr()])),
+                          Create(ref: Ref("indexes"),
+                              params: Obj(["name": "posts_by_name",
+                                           "source": BlogPost.classRef,
+                                           "terms": Arr(Obj(["field": Arr("data", "name")])),
+                                           "values": Arr()]))
+                )
+              , completion: callback)
+          }
+      }
+  }
+}
 
-To install FaunaDB, simply add the following line to your Podfile:
+// Once we've set up fauna scheme we are able to create instances and perform queries among other things...
 
-```ruby
-pod 'FaunaDB', '~> 1.0'
+func createInstances(callback: (Result<Value, Error> -> ())) {
+    client.query({
+        let blogPosts = (1...100).map {
+            BlogPost(name: "Blog Post \($0)", author: "FaunaDB",  content: "content", tags: $0 % 2 == 0 ? ["philosophy", "travel"] : ["travel"])
+        }
+        return Map(collection: Arr(blogPosts)) { blogPost  in
+             Create(ref: Ref("classes/posts"), params: Obj(["data": blogPost]))
+        }
+    }(), completion: callback)
+}
+
+// let's invoke these async functions to set up the scheme and create blog post instances.
+setUpSchema(db_name) { result in
+      guard let _ = try? result.dematerialize() else {
+          // show error message or propagates it
+          return
+      }
+      createInstances { createInstancesR in
+          guard let _ = try? createInstancesR.dematerialize() else { return /* handle error */ }
+          // blog post instances are now saved in FaunaDB
+      }
+}
+
+```
+
+#### Basic Usage (using RxSwift reactive extensions)
```

Comment:
@erickpintor  We can remove rx target (framework) dropping official support out. 

We can provide these 2 methods in the example project just as an example. Will be up to the developer adopt rx extensions by just copying past these helpers.

As you said, It's so simple to implement that it adds some overhead to support it officially, specially to bring support to all platforms.

```swift
// All RxFaunaDB target code
extension FaunaDB.Client {

    public func rx_query(expr: Expr) -> Observable<Value> {
        return Observable.create { [weak self] subscriber in
            let task = self?.query(expr) { result in
                switch result {
                case .Failure(let error):
                    subscriber.onError(error)
                case .Success(let value):
                    subscriber.onNext(value)
                    subscriber.onCompleted()
                }
            }
            return AnonymousDisposable {
                task?.cancel()
            }
        }
    }
}

extension ObservableType where Self.E == Value {

    public func mapWithField<T: DecodableValue where T.DecodedType == T>(field: Field<T>) -> Observable<T> {
        return map { try field.get($0) }
    }
}
```

I suggest to:
* Remove RxFaunaDB framework, driver user can do it easily on his own. 
* Remove any reference to reactive programming from official readme and documentation.
* Since example app is properly splitted up into standalone (pure) and reactive example from the very first view i think it would make sense to keep showing this.